src/transformers/models/gpt_bigcode/modeling_gpt_bigcode.py

```diff
@@ -174,10 +174,9 @@ def _attn(self, query, key, value, attention_mask=None, head_mask=None):
             # The bug was fixed in https://github.com/pytorch/pytorch/pull/96086,
             # but the fix has not been released as of pytorch version 2.0.0.
             attn_weights = torch.zeros_like(attn_weights)
-            beta = 1
+            attn_weights = torch.baddbmm(attn_weights, query, key, beta=0, alpha=scale_factor).view(attn_shape)
```

Comment:
```suggestion
            attn_weights = torch.baddbmm(attn_weights, query, key, beta=1, alpha=scale_factor).view(attn_shape)
```
Small nit to follow the original code ^

Honestly, I'm not sure if we can't just use `matmul` regardless of whether we use cpu or not? Attn_weights are fixed to zero for beta=1 and it seems it was only done because of a cpu bug in `baddbmm` which shouldn't be in `matmul`.

Reply:
Using matmul sounds simpler for sure, I am not even sure we need to pre-init the attn_weights anymore!